packages/core/src/actors/callback.ts

```diff
@@ -231,6 +231,8 @@ export function fromCallback<
         };
 
         callbackState.dispose?.();
+        callbackState.receivers?.clear();
+        instanceStates.delete(actorScope.self);
```

Comment:
I feel it's always better to clear up the data before calling clean up functions (if only possible) - because those functions could potentially throw.
```suggestion
        instanceStates.delete(actorScope.self);
        callbackState.receivers?.clear();
        callbackState.dispose?.();
```